sdk/nodejs/package.json

```diff
@@ -34,13 +33,12 @@
         "@types/semver": "^5.5.0",
         "istanbul": "^0.4.5",
         "mocha": "^3.5.0",
-        "node-gyp": "^3.6.2",
         "tslint": "^5.11.0"
     },
     "pulumi": {
         "comment": "Do not remove. Marks this as as a deployment-time-only package"
     },
     "engines": {
-        "node": ">=8.0.0"
+        "node": ">=8.13.0 || >=10.10.0"
```

Comment:
requirement of grpc-js.  Have to decide if this restriction on nodejs versions is ok.  Note that 8.13 came out 18 months after 8.0 :-/

Reply:
Note that Node 8 is EOL now, and we may be able to drop support entirely soon (https://github.com/pulumi/pulumi/issues/3736).

Reply:
That woudl def be nice :)